kvmagent/kvmagent/keyagent/key_agent_pb2.py

```
@@ -0,0 +1,228 @@
+        if self.description:
+            parts.append(_encode_length_delimited(2, self.description.encode('utf-8')))
+        if self.vm_uuid:
+            parts.append(_encode_length_delimited(3, self.vm_uuid.encode('utf-8')))
+        if self.purpose:
+            parts.append(_encode_length_delimited(4, self.purpose.encode('utf-8')))
+        if self.provider_name:
+            parts.append(_encode_length_delimited(5, self.provider_name.encode('utf-8')))
+        return b''.join(parts)
+
+    @classmethod
+    def FromString(cls, buf):
```

Ruff (0.15.4)

[warning] Unused class method argument: `buf`

(ARG003)